docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/focalmeter/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Kantar FocalMeter"
+sidebar_position: 12000
+---
+
+```mdx-code-block
+import Block5966 from "@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md"
+
+<Block5966/>
+```
+
+This plugin will add integration with the [Kantar FocalMeter](https://www.virtualmeter.co.uk/focalmeter) to your Snowplow tracking.
```

Comment:
Is there a way to reduce duplication between the browser and js versions of this page?

Reply:
This will be addressed separately in #637

Reply:
Have added a reusable component.